open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df_train = pd.read_csv('../../assets/data/train.csv')
</pre><p>where is this file? You need to submit it in the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
It should be put in folder `assets/data/matplotlib-applied/train.csv`. This line of code should be then modified as `df_train = pd.read_csv('../../assets/data/matplotlib-applied/train.csv')`